iterator.py

```diff
@@ -1,19 +1,20 @@
-import os, re, requests
```

Comment:
Imports need to be on separate lines, according to [PEP8](https://www.python.org/dev/peps/pep-0008/)